Zend/tests/bug73663_2.phpt

```diff
@@ -12,7 +12,7 @@ change(list($val) = $array);
 var_dump($array);
 ?>
 --EXPECTF--
-Fatal error: Uncaught Error: change(): Argument #1 ($ref) cannot be passed by reference in %s:%d
+Fatal error: Uncaught Error: change(): Argument #1 ($ref) must be passed by reference in %s:%d
```

Comment:
I don't agree with the wording, the issue *is* that the thing being passed to the function parameter is a value for which a reference cannot be taken, as it's only variables that can be passed by references.

Your wording suggests that I should do something like ``change(&(list($val) = $array))`` which is non sensical.

Better wording may be:  ``Argument #1 ($ref) is a value and therefore cannot be passed by reference in %s:%d``